Cubical/Algebra/CommRing/FGIdeal.agda

```diff
@@ -100,5 +107,146 @@ syntax genIdeal R V = ⟨ V ⟩[ R ]
 
 
 FGIdealIn : (R : CommRing ℓ) → Type (ℓ-suc ℓ)
-FGIdealIn R = Σ[ I ∈ CommIdeal R ] ∃[ n ∈ ℕ ] ∃[ V ∈ FinVec (fst R) n ] I ≡ ⟨ V ⟩[ R ]
-
+FGIdealIn R = Σ[ I ∈ CommIdeal.CommIdeal R ] ∃[ n ∈ ℕ ] ∃[ V ∈ FinVec (fst R) n ] I ≡ ⟨ V ⟩[ R ]
```

Comment:
Couldn't it be easier to work with this if you refactored and had only one `∃`? 

So something like: 

```
FGIdealIn R = Σ[ I ∈ CommIdeal.CommIdeal R ] ∃[ nV ∈ Σ[ n ∈ ℕ ] FinVec (fst R) n ]  ] I ≡ ⟨ nV .snd ⟩[ R ]
```

This way you wouldn't have to eliminate twice and could get n and V by pattern-matching

Reply:
I like that suggestion and copied it. Right now `FGIdealIn` is not used anywhere, but let's keep it and see if it actually ever gets applied...

Reply:
Ah ok, I wonder if there could be some other places where refactoring like this could lead to less uses of `uncurryN`...